netsync/manager.go

```diff
@@ -1926,6 +1927,29 @@ type peerSubscription struct {
 	cancel <-chan struct{}
 }
 
+// ConnectedPeers returns all the currently connected peers to the channel
+// and then any additional new peers on connect.
+func (sm *SyncManager) ConnectedPeers() (<-chan query.Peer, func(), error) {
```

Comment:
I think we should name it to sth else, it's confusing to see three `ConnectedPeers` defined in the repo, maybe `SubscribePeers`.

Reply:
I named it this because we use this function to meet the interface for `query.WorkManager`. Kinda weird to pass in `SubscribePeers` to meet the interface for `ConnectedPeers` imo.

How about naming the other `ConnectedPeers` in main to `SubscribePeers`?